### PR TITLE
SyncUpdates Error when HTTP Property

### PR DESCRIPTION
**Pull request by MiguelAHM**

### CODE REVIEW

File: whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        if (sendError(httpRequest)){
+            final HttpServletResponse httpResponse = (HttpServletResponse)response;
+            httpResponse.sendError(HttpStatus.UPGRADE_REQUIRED_426, "Please switch to HTTPS to continue using HTTPS.");
+            return;
+        }
+
+        chain.doFilter(request, response);
+    }
+
+    private boolean sendError(final HttpServletRequest httpRequest) {
```
Comment (maggarwal13) [inline whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java:34]: Where you are checking if password is there in the request or not?

Comment (MiguelAHM) [reply whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java:34]: The task is just to turn the warn over http into an error
nothing to do with password

Comment (MiguelAHM) [reply whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java:34]: I did rename the property and the title of the PR, maybe it confused you

File: whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+public class SyncUpdatesHttpSchemeFilter implements Filter {
+
+    @Value("${syncupdates.http.error:false}")
+    private boolean passwdError;
```
Comment (maggarwal13) [inline whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java:20]: rename the variable from passwdError to httpSchemeAllowed or something smiliar, if yiu are not checking for password credentials

File: whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import org.eclipse.jetty.http.HttpStatus;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+public class SyncUpdatesHttpSchemeFilter implements Filter {
+
+    @Value("${syncupdates.http.error:false}")
```
Comment (maggarwal13) [inline whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java:19]: may be syncupdates.http.allowed?

Comment (MiguelAHM) [reply whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java:19]: I leave this as it is but changed the variable to
`httpError`

File: whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java (hunk 2 above)
Comment (MiguelAHM) [reply whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java]: yes, I forgot this in the last update

**Review by maggarwal13 — APPROVED** [review]

**Pull request merged**